OpenRA.Mods.Common/Scripting/Global/ReinforcementsGlobal.cs

```diff
@@ -125,11 +125,16 @@ public Actor[] Reinforce(Player owner, string[] actorTypes, CPos[] entryPath, in
 				}
 			}
 
-			for (var i = 1; i < entryPath.Length; i++)
+			// We can't move planes to the last waypoint yet,
+			// as that causes them to teleport to ground level
+			for (var i = 1; i < entryPath.Length - 1; i++)
 				Move(transport, entryPath[i]);
 
+			var lastPosition = entryPath.Last();
```

Comment:
Can you rename this `destination` or `dropoffPoint` or something? `lastPosition` is confusing.